src/reporter.ts

```diff
@@ -0,0 +1,311 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+import SauceLabs from 'saucelabs';
+import { TestRun, Suite, Test } from '@saucelabs/sauce-json-reporter'
+
+const { SummaryFormatter, formatterHelpers, Status } = require('@cucumber/cucumber')
```

Comment:
Better switch this to `import` instead of `require`. You're going to get actual type help if you do.